doc/ref_model/chapters/chapter06.rst

```diff
@@ -210,15 +210,15 @@ These acceleration interfaces are summarized here in Table 6.3 only for convenie
 |                       |                        |       |        | ata           |                                    |
 +-----------------------+------------------------+-------+--------+---------------+------------------------------------+
 | DeRegisterForAccEvent | DeRegisterForAccEventR | VNF → | Input  | accEvent      | Event VNF is deregistering from    |
-| Response              | esponse                | NFVI  |        |               |                                    |
+| Request               | esponse                | NFVI  |        |               |                                    |
```

Comment:
```suggestion
| Request               | Response                | NFVI  |        |               |                                    |
```

Reply:
Pankaj it's incorrectly wrapped (see line above).

is it DeRegisterForAccEvent Response or DeRegisterForAccEventResponse ?
I think many slashes are missing in this tabs.

I think they must be fixed in another PR.

Reply:
@pgoyal01 @walterkozlowski https://github.com/cntt-n/CNTT/issues/3177